### PR TITLE
Remove null values from nodeId Filters

### PR DESCRIPTION
**Pull request by MohamedElmdary**
## Issues
- https://github.com/threefoldtech/grid_weblets/issues/1279

### CODE REVIEW

**Comment by Mahmoud-Emad** [comment]
The issue is still there, I tried the scenario mentioned in the issue, and seems the issue is still there.

**Review by zaelgohary — COMMENTED** [review]
Issue is still there.

image

File: src/components/SelectNodeId.svelte [hunk 1]
```diff
@@ -85,22 +85,29 @@
     }
   }
 
+  const IGNORED_VALUES = [null, "null", "", undefined, "undefined"];
   function onLoadNodesHandler() {
     loadingNodes = true;
     status = null;
     const label = "Please select a node id.";
-    nodeIdSelectField.options[0].label = "Loading...";
+    // nodeIdSelectField.options[0].label = "Loading...";
```
Comment (zaelgohary) [inline src/components/SelectNodeId.svelte]: Can you remove this comment?

**Review by zaelgohary — CHANGES_REQUESTED** [review]

File: src/components/SelectNodeId.svelte [hunk 2]
```diff
@@ -377,7 +384,13 @@
   <h5 class="is-size-5 has-text-weight-bold">Nodes Filter</h5>
   {#each filtersFields as field (field.symbol)}
     {#if nodeSelection === "automatic" || (multiple && field.symbol !== "country")}
-      <Input data={filters[field.symbol]} {field} on:input={_update(field.symbol)} />
+      {filters[field.symbol]}
+      {typeof filters[field.symbol]}
```
Comment (mahmoud-emad) [inline src/components/SelectNodeId.svelte]: Remove those tow lines please if not needed :)

**Review by Mahmoud-Emad — CHANGES_REQUESTED** [review]

**Review by Mahmoud-Emad — APPROVED** [review]

**Review by zaelgohary — APPROVED** [review]

**Pull request merged**